.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ commands:
 jobs:
   build-ruby-2_5:
     docker:
-      - image: circleci/ruby:2.5
+      - image: circleci/ruby:2.7
```

Comment:
I am not sure if it makes much (or rather _any_) sense? 😅

Reply:
Good call. Bots are dumb :-)